problems/countNumbers.js

```diff
@@ -9,8 +9,10 @@
  * returns { 1:3, 2:2, 3:1, 4:1 }
  */
 
-function countNumbers() {
- 
+function countNumbers(arr) {
+    for(let i = 0; i < arr.length; i++)
```

Comment:
Thinking of using a loop here is definitely the right instinct. But we also need to have an object variable and to update the key values as we iterate through it. The keys should be elements in the array, and values should be the number of times we've seen that element.